foxpuppet/browser/navbar.py

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+MENU_BUTTON = 'PanelUI-menu-button'
+
+
+class Navbar(object):
+
+    def __init__(self, selenium, *args, **kwargs):
+        self.selenium = selenium
+        self._locationbar = None
+        self.wait = WebDriverWait(selenium, 10)
```

Comment:
This is a big problem. The reason I am doing this is trying to fix the flakiness that happens in travis. This doesn't work however as I get a timeout exception. I am still having problems with travis when it needs to click a button it seems. It will say that the button I want within the menu button, isn't there.